README.md

```diff
@@ -658,6 +658,33 @@ when you need to retrieve a single record by some attributes.
   # good
   User.where.not(id: id)
   ```
+* <a name="squished-heredocs"></a>
+  When specifying an explicit query in a method such as `find_by_sql`, use
+  heredocs with `squish`. This allows you to legibly format the SQL with
+  line breaks and indentations, while supporting syntax highlighting in many
+  tools (including GitHub, Atom, and RubyMine).
+<sup>[[link](#squished-heredocs)]</sup>
+
+  ```Ruby
+  User.find_by_sql(<<-SQL.squish)
+    SELECT
+      users.id, accounts.plan
+    FROM
+      users
+    INNER JOIN
+      accounts
+    ON
+      accounts.user_id = users.id
+    # further complexities...
+  SQL
+  ```
+
+  `squish` removes the indentation and newline characters so that your server
```

Comment:
Maybe this should be `String#squish` and be a link to the docs (which might outdated, though).